### ISSUE TITLE
Allow to export/import/clone Import Sources, Sync Rules and Jobs

### ISSUE THREAD

**Issue by Thomas-Gelf** [Icinga/icingaweb2-module-director#1409]
This functionality should be available in the GUI, through REST API and on CLI.

**Comment by Thomas-Gelf** [comment]
Update: large parts are already there, mostly export-related ... work is still going on

**Comment by Thomas-Gelf** [comment]
Pushed recent progress, it's now possible to clone Sync Rules in the UI, more to come

**Comment by nbuchwitz** [comment]
Cloning an import source fails with the following stack strace:

```
Uncaught Error: Call to protected method Icinga\Module\Director\Objects\ImportSource::existsWithName() from context 'Icinga\Module\Director\Web\Form\CloneImportSourceForm' in /usr/share/icingaweb2/modules/director/library/Director/Web/Form/CloneImportSourceForm.php:55
Stack trace:
#0 /usr/share/icingaweb2/modules/director/library/vendor/ipl/Html/Form.php(74): Icinga\Module\Director\Web\Form\CloneImportSourceForm->onSuccess()
#1 /usr/share/icingaweb2/modules/director/application/controllers/ImportsourceController.php(137): dipl\Html\Form->handleRequest(Object(Icinga\Web\Request))
#2 /usr/share/icingaweb2/library/vendor/Zend/Controller/Action.php(507): Icinga\Module\Director\Controllers\ImportsourceController->cloneAction()
#3 /usr/share/php/Icinga/Web/Controller/Dispatcher.php(76): Zend_Controller_Action->dispatch('cloneAction')
#4 /usr/share/icingaweb2/library/vendor/Zend/Controller/Front.php(937): Icinga\Web\Controller\Dispatcher->dispatch(Object(Icinga\Web\Request), Object(Icinga\Web\Response))
#5 /usr/shar
```

**Comment by Thomas-Gelf** [comment]
It's mostly there, more related features will be merged after 1.5.0
